tests/D2L.CodeStyle.Analyzers.Test/Specs/LoggingExecutionContextScopeBuilderAnalyzer.cs

```diff
@@ -50,8 +51,8 @@ internal sealed class Foo {
 			{ ConfiguredTaskAwaitable<int> t = m_builder./* LoggingContextRunAwaitable(Use RunAsync for awaitable actions.) */ Run<ConfiguredTaskAwaitable<int>> /**/( ConfiguredTaskTAction ); }
 
 			m_builder./* LoggingContextRunAwaitable(Use RunAsync for awaitable actions.) */ Run /**/( () => TaskAction().ConfigureAwait( false ) );
-			{ ConfiguredTaskAwaitable t = m_builder./* LoggingContextRunAwaitable(Use RunAsync for awaitable actions.) */ Run /**/( TaskAction().ConfigureAwait( false ) ); }
-			{ ConfiguredTaskAwaitable t = m_builder./* LoggingContextRunAwaitable(Use RunAsync for awaitable actions.) */ Run<ConfiguredTaskAwaitable> /**/( TaskAction().ConfigureAwait( false ) ); }
+			{ ConfiguredTaskAwaitable t = m_builder./* LoggingContextRunAwaitable(Use RunAsync for awaitable actions.) */ Run /**/( () => TaskAction().ConfigureAwait( false ) ); }
+			{ ConfiguredTaskAwaitable t = m_builder./* LoggingContextRunAwaitable(Use RunAsync for awaitable actions.) */ Run<ConfiguredTaskAwaitable> /**/( () => TaskAction().ConfigureAwait( false ) ); }

```

Comment:
These were passing the tasks instead of the task-returning func, so didn't actually "compile"